packages/create-astro/src/index.ts

```diff
@@ -10,6 +10,7 @@ import { FRAMEWORKS, COUNTER_COMPONENTS, Integration } from './frameworks.js';
 import { TEMPLATES } from './templates.js';
 import { createConfig } from './config.js';
 import { logger, defaultLogLevel } from './logger.js';
+import { exec } from 'child_process';
```

Comment:
I would recommend to install `execa`, which is used in other Astro packages

Reply:
strong +1, it's already a dep of the project and its been a much better experience IMO.

Reply:
Fair! So although `execa` is a dependency of astro core, it isn't a dependency of `create-astro`. I was just using plain `exec` to cut down on `create-astro`'s install size. Since this is the first taste of the framework, I wanted to keep external deps to a minimum for a faster `npx` experience. I personally don't see a huge benefit to `execa` over `exec` after testing locally; the code and end user output look almost identical (beyond removing that `cd` issue noted below). If we're okay with increasing the bundle size, I'm happy to bring `execa` in! Just noting that it doesn't break much benefit from what I can tell.

Reply:
Check out this list: https://github.com/sindresorhus/execa#why

Every time I've try to use these APIs directly, I eventually run into some bug that execa fixes. Windows is a big one, but also the mixing of `stdout` and `stderr` into a single stream of output (`all`) is something that we want here (I believe npm and other package managers will output some important info to `stderr`).

Reply:
Understood! execa-fying 👍